Kernel/System/Console/Command/Dev/Package/Build.pm

```diff
@@ -138,7 +138,7 @@ sub Run {
         Permission => '644',                                                      # unix file permissions
     );
     if ( !$File ) {
-        $Self->PrintError("File $File could not be written.\n");
+        $Self->PrintError("File $Filename could not be written.\n");
```

Comment:
Hi @Micke90s,

thanks for your contribution. Your fix does eliminate the warning. But I think that it would be more consistent if the complete location would be in the message. Maybe introduce a variable `$Location` and pass that variable both to `FileWrite()` and use it in the message.

Could you provide a new PR for that?